src/frontend/magnify/src/components/rooms/RoomRow/RoomRow.tsx

```diff
@@ -60,6 +70,8 @@ export default function RoomRow({ room, baseJitsiUrl }: RoomRowProps) {
   const modals = useModal();
   const queryClient = useQueryClient();
 
+  const [linkWasCopied, setLinkWasCopied] = useState(false);
```

Comment:
i create an `useNotification()` hook. You can see its usage in LoginForm.tsx. So you can remove this logic